components/molecules/Checkout/SummaryRow.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Text, XStack } from 'tamagui'
+
+interface SummaryRowProps {
+  label: string
+  value: string | number
+  fonts: any
+  color: string
+}
+
+const SummaryRow: React.FC<SummaryRowProps> = ({
+  label,
+  value,
+  fonts,
+  color
+}) => (
+  <XStack>
+    <Text flex={1} fontFamily={fonts.fonts.JetBrainsMonoBold}>
```

Comment:
Improve type declarations for `fonts` to make the code easier to understand and maintain.